server/src/com/cloud/network/NetworkServiceImpl.java

```diff
@@ -852,7 +852,8 @@ public boolean releaseSecondaryIpFromNic(long ipAddressId) {
                 throw new InvalidParameterValueException("Can' remove the ip " + secondaryIp + "is associate with static NAT rule public IP address id " + publicIpVO.getId());
```

Comment:
@niteshsarda I've created a PR for it: https://github.com/apache/cloudstack/pull/1947